src/shared/env.ts

```diff
@@ -35,7 +35,7 @@ if (isBrowser()) {
 
 export const httpBaseInternal = `http://${host}`; // Don't use secure here
 export const httpBase = `http${secure}://${host}`;
-export const wsUriBase = `ws${secure}://${wsHost}`;
+export const wsUriBase = `ws${secure}://${externalHost}`;
```

Comment:
Failing CI, and this would also break dev builds. 

Why are apple devices not running the code in lines 25-26 here, but android is?

Reply:
I'm looking. My initial guess is webkit is just more strict about enforcing the rules on websockets. I'm still pretty new to inferno.  I have a test instance and a mac I can test with if you guys want to send me a image though.

Reply:
I think i see, should be wsUri not wsUriBase on index.tsx. maybe webkit is the only one enforcing it.

Reply:
Gotcha, if u have a mac that'd be helfpul if you could see if that works.

For now i have the 0.16.5 release with the those headers commented out, but let me know if wsUri works.

Reply:
I went and tested it this morning and **wsUri** evaluates to `wss://lemmy:8536/api/v3/ws` because `process.env.LEMMY_WS_HOST` is undefined. So that won't work.

It works if we change `wsUrliBase` to `wsUri` AND update lemmy_ansible to include `- LEMMY_WS_HOST={{ domain }}` under lemmy-ui in **docker-compose.yml**

LEMMY_WS_HOST is not defined anywhere.

Reply:
Could you try this line? Replace L32 with:

`wsHost = process.env.LEMMY_WS_HOST || externalHost;`

Reply:
> `wsHost = process.env.LEMMY_WS_HOST || externalHost;`

That appears to work yes.